gschem/src/x_compselect.c

```diff
@@ -79,7 +79,7 @@ compselect_get_view (Compselect *compselect)
     case 1: return VIEW_CLIB;   /* Component library page */
     default:
       g_critical ("compselect_get_view: Unknown tab position\n");
-      return 0;
+      return (enum compselect_view) 0;
```

Comment:
I suggest returning a specific member of `enum compselect_view` here, instead of using a cast.  I suggest `VIEW_CLIB`, since that's the default page in the component selector.  Since this is technically an error pathway, it doesn't matter what you return as long as it's a sensible default.